libsplinter/src/admin/rest_api/mod.rs

```diff
@@ -90,13 +90,12 @@ impl RestResourceProvider for AdminService {
 /// * `rest-api-actix`
 #[derive(Clone)]
 pub struct CircuitResourceProvider {
-    node_id: String,
     store: Box<dyn AdminServiceStore>,
 }
 
 impl CircuitResourceProvider {
-    pub fn new(node_id: String, store: Box<dyn AdminServiceStore>) -> Self {
-        Self { node_id, store }
+    pub fn new(store: Box<dyn AdminServiceStore>) -> Self {
```

Comment:
Note this change is pub and should not be backported.

Reply:
Updated the commit message to say that